# Deliver crate to a person

# Define the goal
string crate_id
geometry_msgs/Point crate_position
---
# Define the result
bool crate_delivered
---
# Define a feedback message
# No intermediate feedback
